example/src/main/java/org/dst/drpc/pb/PBServerImpl.java

```diff
@@ -0,0 +1,56 @@
+package org.dst.drpc.pb;
+
+import org.dst.drpc.pb.generated.CommonProtocol;
+import org.dst.drpc.pb.generated.StringProtocol;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class PBServerImpl implements IPBServer {
+
+  private Logger logger = LoggerFactory.getLogger(getClass());
+  private ExecutorService executorService = Executors.newFixedThreadPool(32);
+
+  @Override
+  public CompletableFuture<StringProtocol.GetResponse> get(StringProtocol.GetRequest request) {
+    CompletableFuture<StringProtocol.GetResponse> future = new CompletableFuture<>();
+    StringProtocol.GetResponse response = StringProtocol.GetResponse.newBuilder()
+          .setValue(request.getKey() + " get() request success")
+          .setStatus(CommonProtocol.Status.OK)
+          .build();
+    //exec sth delayed async
+    executorService.submit(() -> {
+      sleep(3000);
+      future.complete(response);
+    });
+    System.out.println("server receive: "+request.getKey());
+    return future;
+  }
+
+  @Override
+  public CompletableFuture<StringProtocol.PutResponse> put(StringProtocol.PutRequest request) {
+    CompletableFuture<StringProtocol.PutResponse> future = new CompletableFuture<>();
+    StringProtocol.PutResponse response = StringProtocol.PutResponse.newBuilder()
+          .setStatus(CommonProtocol.Status.OK)
+          .build();
+    //exec sth delayed async
```

Comment:
Please use multiple services in this server since this [PR](https://github.com/dst-project/drpc/pull/29) has merged.